examples/demo-stacks/.ecrc

```diff
@@ -0,0 +1,38 @@
+# EditorConfig for Terraform repository
```

Comment:
Rename to the more conventional filename of .editorconfig

Reply:
Done